### PR TITLE
Add description how to run on ESP32-S3-WROOM-1

### PR DESCRIPTION
**Pull request by bartoszcisek**
- extended readme
- added partition table definition required for smaller (32mb) flash

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,6 +49,102 @@ To start using the KPPTR-firmware and exploring its capabilities, follow these s
...
+2. Set flash size to 16mb in project settings: `Partition table -> custom partition CSV file -> partitions-16mb.csv` 
+3. Quit menu saving changes
+4. Rebuild project
+
+#### Flashing firmware
+1. Attach ESP32 board with USB cable to computer. `pwr` led should turn on.
+2. Holding `rst` button press once `boot` button. This will initiate programming sequence on ESP32 side.
+2. Find how your system recognise USB attached ESP32 board with command
+    ```bash
+    $ ls -l /dev/cu.*
```
Comment (bartekM-pl) [inline README.md:127]: Serial port naming "cu.*" works only on OSX. Can you add variant for Linux and Windows? Windows command will be tricky.

File: README.md [hunk 2]
```diff
@@ -49,6 +49,102 @@ To start using the KPPTR-firmware and exploring its capabilities, follow these s
...
+    crw-rw-rw-  1 root  wheel  0x9000001 Nov 26 21:06 /dev/cu.usbmodem21401
+    ```
+4. Use USB device to flash firmware using command:
+    ```bash
+    $ idf.py -p /dev/cu.usbmodem21401 -b 9600 flash monitor
```
Comment (bartekM-pl) [inline README.md]: Don't use specific serial port name. Use something generic like serial_port_name etc.

File: README.md [hunk 3]
```diff
@@ -49,6 +49,102 @@ To start using the KPPTR-firmware and exploring its capabilities, follow these s
...
+
+#### ESP32-S3-WROOM-1-N16R8
+One of them is `ESP32-S3-WROOM-1-N16R8`. It can be flashed directly via USB cable without additional programmer. Only two
+adjustments needs to be done. Run in `PTR_mega_firmware` directory command `idf.py menuconfig` and using menu:
+1. Set partition table definition to match 16MB flash: `Serial flasher config -> flash size -> 16MB`
+2. Set flash size to 16mb in project settings: `Partition table -> custom partition CSV file -> partitions-16mb.csv` 
+3. Quit menu saving changes
+4. Rebuild project
+
+#### Flashing firmware
```
Comment (bartekM-pl) [inline README.md]: Change #### to ##. This should be separated section.

**Review by bartekM-pl — CHANGES_REQUESTED** [review]
Few changes needed but looks promising.

File: README.md (hunk 1 above)
Comment (bartoszcisek) [reply README.md:127]: I don't have machine with Windows. Added MacOS and Linux.

File: README.md (hunk 2 above)
Comment (bartoszcisek) [reply README.md]: Done.

**Comment by bartoszcisek** [comment]
Rebased and added fixes after review.

**Review by bartekM-pl — APPROVED** [review]

**Pull request merged**